docs/command-reference.md

```diff
@@ -0,0 +1,632 @@
+# Container CLI Command Reference
+
+## Core Commands
+
+### `container run`
+
+Runs a container from an image. If a command is provided, it will execute inside the container; otherwise the image's default command runs. By default the container runs in the foreground and STDIN remains closed unless `-i`/`--interactive` is specified.
+
+**Usage**
+
+```bash
+container run [OPTIONS] IMAGE [COMMAND] [ARG...]
+```
+
+**Options**
+
+*   **Resource management**
+    *   `-c, --cpus <number>`: number of CPUs to allocate
+    *   `-m, --memory <size>`: memory limit (K/M/G suffixes)
+*   **Process control**
+    *   `-e, --env <key=value>`: set environment variables
+    *   `--env-file <file>`: read environment variables from a file
+    *   `-i, --interactive`: keep STDIN open for interactive processes
+    *   `-t, --tty`: allocate a pseudo-TTY
+    *   `-u, --user <user>`: run as the specified user name or UID
+    *   `--uid <uid>`: run as the specified numeric user ID
+    *   `--gid <gid>`: run as the specified numeric group ID
+    *   `--cwd <path>`: set the working directory inside the container
+*   **Container management**
+    *   `--name <name>`: assign a name to the container
+    *   `-d, --detach`: run the container in the background (daemon mode)
+    *   `--rm`: automatically remove the container when it exits
+    *   `--cidfile <file>`: write the container ID to a file
+*   **Registry**
+    *   `--scheme <scheme>`: registry scheme (`auto`, `http`, or `https-`)
+*   **Progress**
+    *   `--disable-progress-updates`: disable progress display for image pulls and pushes
```

Comment:
I think this isn't the right description. This disables the progress animation for updates. That could include other things than image pulls and pushes.